src/pow.cpp

```diff
@@ -147,6 +219,41 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, int64_t nF
     return bnNew.GetCompact();
 }
 
+/** Perform the fork difficulty reset */
+unsigned int CalculateMVFResetWorkRequired(const CBlockIndex* pindexLast, int64_t nFirstBlockTime, const Consensus::Params& params)
+{
+    const arith_uint256 bnPowLimit = UintToArith256(params.powLimit); // MVF-BU moved here
+
+    arith_uint256 bnNew, bnNew1, bnNew2, bnOld;
+
+    // drop difficulty via factor
+    int nDropFactor = 4;
```

Comment:
Suggest to abstract it out to a function which takes the difficulty of the previous block and the consensus params.

The function can then apply a drop factor or return a suitable fixed difficulty, as we choose.
Will make it easy for anyone to adjust.